server/models/migrations/next/migrate.sql

```diff
@@ -47,3 +47,87 @@ UPDATE unit SET `name` = 'Analysis of Cashbox', `key` = 'REPORT.ANALYSIS_AUX_CAS
  */
 INSERT IGNORE INTO unit VALUES
   (307, 'Asset Management', 'TREE.ASSET_MANAGEMENT.TITLE', 'Asset Management', 0, '/ASSET_MANAGEMENT_FOLDER');
+
+/*
+ * @author: mbayopanda
+ * @description: Shipment tables
+ * @date: 2022-02-07
+ */
+DROP TABLE IF EXISTS `shipper`;
+CREATE TABLE `shipper` (
+  `id` SMALLINT(5) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(100) NOT NULL,
+  PRIMARY KEY (`id`)
+ ) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
+
+DROP TABLE IF EXISTS `shipment_status`;
+CREATE TABLE `shipment_status` (
+  `id` SMALLINT(5) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(100) NOT NULL,
+  `translation_key` VARCHAR(100) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
+
+DROP TABLE IF EXISTS `shipment`;
+CREATE TABLE `shipment` (
+  `uuid`                      BINARY(16) NOT NULL,
+  `project_id`                SMALLINT(5) UNSIGNED NOT NULL,
+  `reference`                 INT(11) UNSIGNED NOT NULL,
+  `name`                      VARCHAR(100) NOT NULL,
+  `description`               TEXT NULL,
+  `note`                      TEXT NULL,
+  `shipper_id`                SMALLINT(5) UNSIGNED NOT NULL,
+  `requisition_uuid`          BINARY(16),
+  `origin_depot_uuid`         BINARY(16),
+  `current_depot_uuid`        BINARY(16),
+  `destination_depot_uuid`    BINARY(16),
+  `anticipated_delivery_date` DATE,
+  `date_sent`                 DATE,
+  `date_delivered`            DATE,
+  `receiver`                  VARCHAR(100),
+  `status_id`                 SMALLINT(5) UNSIGNED NOT NULL,
+  `created_by`                SMALLINT(5) UNSIGNED NOT NULL,
+  `created_at`                TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `updated_by`                SMALLINT(5) UNSIGNED NULL,
+  `updated_at`                TIMESTAMP NULL,
```

Comment:
This should have ON UPDATE CURRENT_TIMESTAMP or something similar.

Reply:
RESOLVED 👍🏽